example_contracts/addr2uint256.sol

```diff
@@ -0,0 +1,10 @@
+pragma solidity ^0.8.14;
```

Comment:
I don't think we need this test case in here, since it's also being added in the behavior tests. The expectation is for it to be `'Successful'`, the behavior test already checks it doesn't fail at transpilation or compilation, so this would be checked twice. Other than that, it looks good to me.

Reply:
Yeah, Github CI runs will check this twice but the idea is to do syntax check on the output when a person only runs `bin/warp test` or if they only run`yarn test:behaviour`